from .handler import CustomException


class CodeException(CustomException):
    pass
